File: test_cases/Test2.c
#include <stdio.h>
#include <stdbool.h>

float funzione(int x, float y, char z){

    int a = 4;
    float b = 6.35;

    b = a + b * 4;
    printf("%d", x);
    printf("%f", y);
    printf("%c", z);

    return b;
}

int main() {

    int x = 5;
    float y = 3.65;
    char z = 'g';

    x = funzione(y,z,x);
    printf("%d", x);

    return 0;
}

File: test_cases/Test2.py
#Source generato dal frontend C->Python

def funzione( x,y,z):
	 a = 4
	 b = 6.35
	 b = (a + (b * 4))
	 print ( x )
	 print ( y )
	 print ( chr( z ) )
	 return b
def main( ):
	 x = 5
	 y = 3.65
	 z = ord('g')
	 x = int( funzione( int(y), z, x ) )
	 print ( x )
	 return 0

main()